hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterServiceImpl.java

```diff
@@ -1285,7 +1293,7 @@ private void sendMembershipEventNotifications(MemberImpl member, Set<Member> mem
             final MembershipServiceEvent event = new MembershipServiceEvent(membershipEvent);
             for (final MembershipAwareService service : membershipAwareServices) {
                 // service events should not block each other
```

Comment:
If all events are consumed by 1 threaded executor, don't we making this comment useless?
With this PR, as far as i understand, a service should wait event processing of a previous service in executors queue.